src/pickit.py

```diff
@@ -27,6 +28,11 @@ def pick_up_items(self, char: IChar) -> bool:
         found_items = False
         keyboard.send(self._config.char["show_items"], do_press=True, do_release=False)
         time.sleep(1.0) # sleep needed here to give d2r time to display items on screen on keypress
+        #Creating a screenshot of the current loot
+        if self._config.general["loot_screenshots"]:
+            img = self._screen.grab()
+            cv2.imwrite("./loot_screenshots/info_debug_drop_" + str(time.time()) + ".png", img)
```

Comment:
Ah see, there you added a timestamp. I would suggest having a more readable one e.g. `time.strftime('%X-%x')`. This way it is easier to cross-check with the log.

Reply:
yes thats my thoght also but didnt found out so fast how to add better timestamp :D